### PR TITLE
fix: update managed deps through package manager

### PR DESCRIPTION
**Pull request by exKAZUu**
## Customer Summary

- `wbfy` now updates its managed JavaScript tooling dependencies through the target package manager instead of a stale internal baseline value.
- Managed dependency checks and updates now respect Yarn/Bun package age restrictions.
- Repository package files were regenerated with the updated managed dependency versions.

## Technical Summary

- Removed `packages/wbfy/package.json` as the managed dependency version source.
- Kept the managed dependency name list, but changed real installs to call `yarn add` / `bun add` with bare package names so the package manager selects the allowed version.
- Existing managed dependencies in either `dependencies` or `devDependencies` are now included in the update pass even when they are not otherwise newly added by `wbfy` conventions.
- Shared the generated Yarn/Bun age-gate settings with package-json generation so update decisions and `--skip-deps` writes use the same constraints.
- Avoided writing `*` over existing pins when npm lookup fails, and avoided prerelease pins for ordinary age-gated dependencies in `--skip-deps` mode.
- Updated brittle tests so they assert externally visible behavior instead of specific tool versions, including the Gemini-requested whitespace-only parser case.
- Re-ran `wbfy`, updating managed tool versions across package manifests and `yarn.lock`.

## Why

- The previous baseline-value behavior let `wbfy` fall behind npm latest and prevented managed dependencies such as `@willbooster/wb` from being updated by re-running `wbfy`.
- Letting the package manager resolve managed dependency updates preserves the managed dependency scope while honoring Yarn/Bun age gates and lockfile behavior.
- The review pass found edge cases where the package-json-only path could disagree with package-manager resolution; those paths now use age-gated stable resolution.

## Testing

- `yarn vitest run test/packageJson.test.ts` from `packages/wbfy`
- `yarn vitest run test/parseCommandLineArgs.test.ts` from `packages/shared-lib`
- `yarn start /Users/exkazuu/ghq/github.com/WillBooster/shared` from `packages/wbfy`
- `yarn verify`
- `bunx @willbooster-private/agentic-workflows@latest skills review --agent all`
- resolved all PR review threads
- pre-commit cleanup and lint hooks
- pre-push workspace lint hooks
- PR CI

### CODE REVIEW

File: packages/shared-lib/test/parseCommandLineArgs.test.ts [hunk 1]
```diff
@@ -27,19 +20,7 @@ test('parseCommandLineArgs with mixed quotes', () => {
...
-  expect(parseCommandLineArgs('--flag=value -f value --option="quoted value"')).toEqual([
-    '--flag=value',
-    '-f',
-    'value',
-    '--option=quoted value',
-  ]);
+test('parseCommandLineArgs ignores empty space-only arguments', () => {
+  expect(parseCommandLineArgs(' arg1  arg2   arg3 ')).toEqual(['arg1', 'arg2', 'arg3']);
+  expect(parseCommandLineArgs('')).toEqual([]);
 });
```
[the remark this answers is not in the document]
Comment (exKAZUu) [reply packages/shared-lib/test/parseCommandLineArgs.test.ts:27]: Fixed in the latest branch state: the whitespace-only assertion `expect(parseCommandLineArgs('   ')).toEqual([]);` is now included in `parseCommandLineArgs ignores empty space-only arguments`, and the focused parser test passes.
